src/surface_tracker/visual_anchors.py

```diff
@@ -0,0 +1,87 @@
+import typing as T
+
+import numpy as np
+import cv2
+
+from .camera import CameraModel
+from .corner import CornerId
+from .location import SurfaceLocation
+from .surface import Surface, SurfaceId
+
+
+class SurfaceVisualAnchors:
+    @staticmethod
+    def _create_from_location(
+        location: SurfaceLocation, camera_model: CameraModel
+    ) -> "SurfaceVisualAnchors":
+
+        perimeter_corners = CornerId.all_corners()
+        perimeter_corners = [c.as_tuple() for c in perimeter_corners]
+        perimeter_corners.append(perimeter_corners[0])
+
+        top_indicator_corners = [[0.3, 0.7], [0.7, 0.7], [0.5, 0.9]]
+        top_indicator_corners.append(top_indicator_corners[0])
+
+        perimeter_points_surface_space = np.array(perimeter_corners, dtype=np.float32)
+        perimeter_points_image_space = location._map_from_surface_to_image(
+            points=perimeter_points_surface_space,
+            camera_model=camera_model,
+            compensate_distortion=False,
+        )
+
+        top_indicator_points_in_surface_space = np.array(
+            top_indicator_corners, dtype=np.float32
+        )
+        top_indicator_points_in_image_space = location._map_from_surface_to_image(
+            points=top_indicator_points_in_surface_space,
+            camera_model=camera_model,
+            compensate_distortion=False,
+        )
+
+        title_anchor = perimeter_points_image_space.reshape((5, -1))[2]
+        title_anchor = title_anchor[0], title_anchor[1] - 75
+
+        edit_surface_anchor = title_anchor[0], title_anchor[1] + 25
+        edit_markers_anchor = title_anchor[0], title_anchor[1] + 50
+
+        return SurfaceVisualAnchors(
```

Comment:
Why not make it a classmethod and do this?
```suggestion
        return cls(
```